spec/core/data_structures.md

```diff
@@ -124,22 +124,22 @@ The steps to validate a new block are:
-| EvidenceHash      | slice of bytes (`[]byte`) | MerkleRoot of the evidence of Byzantine behavior included in this block.                                                                                                                                                                                                                                                                                                             | Must  be of length 32                                                                                                                                                                            |
-| ProposerAddress   | slice of bytes (`[]byte`) | Address of the original proposer of the block. Validator must be in the current validatorSet.                                                                                                                                                                                                                                                                                         | Must  be of length 20                                                                                                                                                                            |
+| Name              | Type                      | Description                                                                                                                                                                                                                                                                                                                                                                            | Validation                                                                                                                                                                                                                                                 |
+|-------------------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
```suggestion
|-----|------|-----|-----|
```

Reply:
Not sure if I understood.

Reply:
Just a suggestion.
These dashes just clutter the file, but have not effect on the rendered table.